quay_xfer.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+### requires jq + skopeo to be installed locally
+
+# arg check
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 source_namespace/source_repo dest_namespace/dest_repo"
+  exit 1
+fi
+
+SOURCE_REPO="$1"
+DEST_REPO="$2"
+
+# quay credentials
+USERNAME=''
+PASSWORD=''
```

Comment:
I'm guessing these need to be manually added to the script?

Reply:
Yes but do you know of any examples of how to properly handle creds in a script like this? I feel like this is a big no-no from prodsec as an easy way to accidentally push credentials up but also passing as args felt problematic.

Reply:
Would running `skopeo login` before executing the script be enough? That at least feels like a more secure way to do it.

Reply:
Interesting I will try that workflow and see if i can remove it from the script

Reply:
It's also possible to check if you're already logged in to a repository by running e.g. `skopeo login --get-login quay.io` which should return the name of the logged in user or an error.

Reply:
I've been playing around with this; the login check is a useful bit but the script needs a little massaging to get things working gracefully.
I think what will work well is to check against the target and destination repos for a current valid auth and if none is found it will ask for login credentials from the user prompt.